third_party/amd/lib/TritonAMDGPUToLLVM/Utility.cpp

```diff
@@ -189,16 +393,37 @@ Value llGetPid(Location loc, RewriterBase &rewriter, ModuleOp moduleOp,
 }
 
 Value llLoad(RewriterBase &rewriter, Location loc, Value ptr, Type elemTy,
-             Value pred, Value falseVal, int64_t alignmentBytes,
-             triton::CacheModifier cm) {
-
-  // Try to emit llvm.intr.masked.load if we can. In theory the backend should
-  // be happier because we emit less branchy code to optimize. The backend will
-  // lower it down however it wants at some point.
-  if (alignmentBytes &&
-      (cm == triton::CacheModifier::CG || cm == triton::CacheModifier::NONE)) {
-    // `llvm.intr.masked.load` only accepts vectors. If we see a scalar we need
-    // to bitcast to `vector<1xelemTy>` (and back)
+             Value pred, Value falseVal,
+             std::optional<triton::AMD::TargetInfo> targetInfo,
+             int64_t alignmentBytes, triton::CacheModifier cm,
+             bool useBufferOps) {
+
+  bool noCacheModifiers = (cm == triton::CacheModifier::NONE);
+  bool nt = (cm == triton::CacheModifier::CG);
```

Comment:
Nit: prefer to add a `// Non temporal` at the end to be clear. `nt` is certainly not very clear for folks not deep in this domain but still trying to read the code (esp it comes from `::CG`). :)

Reply:
I renamed the variable as `nonTemporal`.